### ISSUE TITLE
can't satisfy 'octicons (~> 8.5)', already activated 'octicons-12.0.0'

### ISSUE THREAD

**Issue by DanSman** [gollum/gollum#1672]
Hey

I've just update to the new version. Not sure that's really a bug on your side but I only use ruby for gollum so I thought I'll try my best here.

After the update I get the following problem:
```
Feb 17 12:58:27 laptop bash[113552]: /usr/lib/ruby/2.7.0/rubygems/specification.rb:1396:in `block in activate_dependencies': can't satisfy 'octicons (~> 8.5)', already activated 'octicons-12.0.0' (Gem::LoadError)
Feb 17 12:58:27 laptop bash[113552]:         from /usr/lib/ruby/2.7.0/rubygems/specification.rb:1388:in `each'
Feb 17 12:58:27 laptop bash[113552]:         from /usr/lib/ruby/2.7.0/rubygems/specification.rb:1388:in `activate_dependencies'
Feb 17 12:58:27 laptop bash[113552]:         from /usr/lib/ruby/2.7.0/rubygems/specification.rb:1370:in `activate'
Feb 17 12:58:27 laptop bash[113552]:         from /usr/lib/ruby/2.7.0/rubygems.rb:302:in `block in activate_bin_path'
Feb 17 12:58:27 laptop bash[113552]:         from /usr/lib/ruby/2.7.0/rubygems.rb:301:in `synchronize'
Feb 17 12:58:27 laptop bash[113552]:         from /usr/lib/ruby/2.7.0/rubygems.rb:301:in `activate_bin_path'
Feb 17 12:58:27 laptop bash[113552]:         from /home/dan/.gem/ruby/2.7.0/bin/gollum:23:in `<main>'
```

I tried v8.5 and 12 both aren't working. 

Is there anything else I could do?

Thanks

**Comment by dometto** [comment]
Ah, hope I didn't break this in the new version!

Are you running from source with Bundler? In that case try removing
Gemfile.lock and installing again?

**Comment by DanSman** [comment]
Where can I find the file?

I installed it like [your wiki](https://github.com/gollum/gollum#installation).

**Comment by DanSman** [comment]
Hm.

Maybe I leave it open. `gem clean` deletes
```
Successfully uninstalled gollum-lib-5.0.6
```
which gives me the error message.

**Comment by dometto** [comment]
Sorry, it looks like there is a bit of a dependency mess. The current version of gollum on rubygems will install `gollum-lib 5.1`, but that depends on a newer `octicons`. I released the new gollum-lib because currently development PRs on gollum were stalled by failing tests. This difficulty will persist until we release the corresponding new version of gollum itself, which hopefully will be very soon. For now, if you want to make sure it works, just install the previous version of gollum-lib by hand:

`gem install gollum-lib -v 5.0.5`

**Comment by DanSman** [comment]
No problem.  
I just keep updating/cleaning as usual and install it again. That's fine. I'm glad we found a solution and I can continue to use it :)  
  
So I'm going to leave this issue open until it's resolved, right?

**Comment by dometto** [comment]
Just released gollum `v5.2`! That should fix this issue. For anyone running into it: just uninstall `gollum` and `gollum-lib` and then `gem install gollum` again.
